src/adapters/appnexus.js

```diff
@@ -186,6 +185,7 @@ AppNexusAdapter = function AppNexusAdapter() {
         bid.width = jptResponseObj.result.width;
         bid.height = jptResponseObj.result.height;
         bid.dealId = jptResponseObj.result.deal_id;
+        bid.bidId = id;//set bidId, so later on the correct response can be matched with the request, if AN is in the race multiple times for the same adUnit
```

Comment:
As mentioned elsewhere this is achieved in #477 by passing the origin bid request to `bidfactory.createBid`.